db/migrate/20200224095152_default_nil_fields_end_user_removal.rb

```diff
@@ -0,0 +1,13 @@
+class DefaultNilFieldsEndUserRemoval < ActiveRecord::Migration[5.0]
```

Comment:
Shouldn't we do it separately from this PR? Prepare the database so it can accept null values then remove from code the usage? Maybe I am missing missing something 🤔

Reply:
I agree

Reply:
Boom #1734